### PR TITLE
add ssh info to dataproc 'dashboard', added a couple of images...

### PR DESCRIPTION
**Pull request by amygdala**
… other minor cleanup

### CODE REVIEW

**Comment by amygdala** [comment]
@johntbates , would you have time to take a look?  This adds some info on ssh key setup and also gives the 'clone' command in a later step as a fallback in case the user doesn't have a GH account.

I also added a couple images whose public URLs will be used in a subsequent PR.

File: dataproc/hail_demo_dashboard.md [hunk 1]
```diff
@@ -8,7 +8,17 @@ The example specifically annotates GWAS results from the paper [*Whole genome se
 
 # How do I get started?
 
-*Duplicate* this workspace. You can do that via the 'three-dot' menu in the upper right of the workspace.
+## Step 0: Set up your github SSH key and _Duplicate_ this workspace
+
+- In the Enterprise Terra UI, click on your profile icon, then click on "My Profile". Copy the public SSH key listed there.
+  Then, [add that key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) to your GitHub account.
+  This page: https://terra-docs.api.verily.com/docs/how_to_guides/terra_ssh_key_guide/ has more detail.
```
Comment (johntbates) [inline dataproc/hail_demo_dashboard.md]: I'd make this a markdown link.  e.g. 

See [this page](https://terra-docs.api.verily.com/docs/how_to_guides/terra_ssh_key_guide/) for a more details.

**Review by johntbates — APPROVED** [review]

File: dataproc/hail_demo_dashboard.md (hunk 1 above)
Comment (amygdala) [reply dataproc/hail_demo_dashboard.md]: done

**Comment by amygdala** [comment]
thanks @johntbates !

**Pull request merged**